CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - [RB-184] Add support for abort signal
 - [RB-205] Add support for unit selection
+- [RB-210] Add support for vehicle attachments
```

Comment:
Oh dear! I forgot to change this Unreleased to be 0.5.1. So your new one would be out of this block sorry